users/uuid/idp.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Mainflux
```

Comment:
Is the same one used for things (like UUIDv4 generator)? Maybe then can be moved to project root and mutualized between two packages?

Reply:
I agree, I wanted to propose this also. Especially because there are 3 services that use idp

Reply:
I'll fix in a separate PR: https://github.com/mainflux/mainflux/issues/1154